.github/workflows/atlas-image-build.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      - uses: tj-actions/changed-files@v38
+      - uses: tj-actions/changed-files@v39
```

Comment:
@pavannd1 As a separate task, we should move these tags to immutable sha256 references.

Reply:
```suggestion
      - uses: tj-actions/changed-files@48566bbcc22ceb7c5809ebdd27377309f2c3de8c # v39
```